modin/distributed/dataframe/pandas/partitions.py

```diff
@@ -78,6 +128,13 @@ def unwrap_partitions(
         )
 
     modin_frame = api_layer_object._query_compiler._modin_frame
+
+    is_modin_distributed = hasattr(modin_frame, "_partition_mgr_cls")
+
+    if not is_modin_distributed and axis == 0:
+        # Directly uses the iloc API to extract data.
+        return _ray_from_modin_non_partition(api_layer_object=api_layer_object)
```

Comment:
Hmm, I think I have to ask again - what if user activated Dask as current engine? This is regarding the docstring of the outer function:
```
    Returns
    -------
    list
        A list of Ray.ObjectRef/Dask.Future to partitions of the ``api_layer_object``
        if Ray/Dask is used as an engine.
```
I wonder if we should augment the check on line 134 with `Engine.get() == 'Ray'`?..